posixlib/src/main/scala/scala/scalanative/posix/unistd.scala

```diff
@@ -49,6 +49,34 @@ object unistd {
              path2: CString,
              flag: CInt): CInt = extern
 
+  // NULL, see POSIX stddef
```

Comment:
The comment is good, but this is another case of you are posix or not.  posix defines null in unistd.h, I just looked.  At the least, the NULL here should be a reference
to the NULL in stddef (or visa versa).  That would make each file (more) posix compliant and reduce the amount of SN lore a user of SN needs to know.

Reply:
I was going to just add it but the type is weird on macOS so I avoided it.

NULL is there but extra info is in stddef - this is a note to future devs.